### DISCUSSION TITLE
Sprite tearing in level editor

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#13670] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I noticed that two of the ruin back wall sprites tear when you try to cover a large area with them, but only in level editor. Mildly annoying. Initially I wrote this report because I thought the sprites were broken outside of the level editor as well, but testing shows that is not the case.
![RuinWallTexture1](https://github.com/FakeFishGames/Barotrauma/assets/47554589/ab807e70-ecdf-4cf3-b415-574515618808)
![RuinWallTexture2](https://github.com/FakeFishGames/Barotrauma/assets/47554589/b641f6cc-df49-4dd8-a8d3-d3a8e18ab295)
![RuinWallTextureView](https://github.com/FakeFishGames/Barotrauma/assets/47554589/4d0c4fce-6719-4187-8a6f-50458eac79a6)
![RuinWallTextureView2](https://github.com/FakeFishGames/Barotrauma/assets/47554589/f0ebbec4-4b08-4ca5-86df-bdc4a5840cc4)

### Reproduction steps
Tearing on these sprites does not happen in submarine editor or in game.
It only appeared after I generated a level with these sprites in the level editor. That's where the screenshots come from.
Here is a structure I was working on at the time (the package features .sub files and outpost generation parameters). To reproduce the issue, you can install the package and generate "endoutpostmid" in the level editor.
[EndingRedesign.zip](https://github.com/FakeFishGames/Barotrauma/files/14757764/EndingRedesign.zip)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by BlackMoons** [comment]
I suspect these are floating point bugs caused by the way barotrauma tiles textures (issuing separate internal draw calls and geometry)
